server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -1070,7 +1068,7 @@ public boolean isRemoteStoreEnabled() {
      * Returns if remote translog store is enabled for this index.
      */
     public boolean isRemoteTranslogStoreEnabled() {
-        return isRemoteTranslogStoreEnabled;
+        return isRemoteStoreEnabled;
```

Comment:
nit : Should we add a comment here for the assumption that if remote store is enabled RTS would be enabled as well?

Reply:
Sure, that would be helpful. will add